vendor/engines/secure_rooms/app/controllers/secure_rooms/indala_numbers_controller.rb

```diff
@@ -0,0 +1,54 @@
+module SecureRooms
+
+  class IndalaNumbersController < ApplicationController
+
+    include TextHelpers::Translation
+
+    customer_tab :password
+    admin_tab     :all
```

Comment:
Rubocop/Codeclimate is complaining about this. A lot of existing code still uses argument alignment like this, but we're moving more towards not aligning arguments, so I would recommend closing up the spacing here.